### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by Lokesh-Krishna**
This is with reference to #308 
I intend to update this again once we update to the new icon and the new color scheme.
Is there anything more I need to add? On my device (OnePlus One running CyanogenMod 14.1) I can't see the app requesting any permissions at all. Are there any permissions that this app does request? The lack of permissions is something that I have mentioned as a feature.

Edited by @di72nn:
 - [ ] Squash commits before merge.

### CODE REVIEW

**Comment by Lokesh-Krishna** [comment]
Um, I am no developer so I don't understand the pending check. Can anybody help me out with that?

File: README.md [hunk 1]
```diff
@@ -1,24 +1,57 @@
...
+The android app lets you   
+- Connect to your self hosted wallabag instance or connect to your Framabag.  
+- Increase and decrease the size of the font and also switch between a serif or sans-serif font for a more comfortable reading experience.
+- Switch between numerous themes.  
+- Needs no special permissions.
+
+There are a lot more feautures that we are working on currently and we are looking forward to bringing them to you, including support for wallabag 2.0 and a redesign of the app to bring it even more in alignment with Google's Material Design language.
+
+## Screenshots
+<img src="https://github.com/Lokesh-Krishna/android-app/blob/master/Wallabag%20Reading%20List.png" align="left"
```
Comment (nicosomb) [inline README.md]: I'd prefer to host these pictures in our repository.

**Comment by di72nn** [comment]
[Preview.](https://github.com/Lokesh-Krishna/android-app/blob/Lokesh-Krishna-patch-1/README.md)
@Lokesh-Krishna thanks for your work :)
There are some things that caught my eye:
 * Make screenshots clickable (nothing fancy, just link to the image file in the repository).
 * If you mention framabag, you should probably also mention https://wallabag.it.
 * The app does support wallabag v2, but limited in features.
 * "GPL licence" and "MIT licence" are not specific enough. There are different versions of each, so if you do mention a license, make it specific.
 * "Material Design ~~language~~guidelines".
 * As @nicosomb noted, use relative URLs for files in the repository (`Wallabag%20Reading%20List.png` instead of `https://github.com/Lokesh-Krishna/android-app/blob/master/Wallabag%20Reading%20List.png`).
 * Permissions. Prior to Android 6 the app requires "full network access", "view network connections" and "run at startup" (the last one is required for auto-sync and is not used if auto-sync is not turned on). Starting with Android 6 the system requests permissions at app's runtime (but these particular permissions are not considered dangerous so system grants them without requests). Though, I'm not very familiar with permission handling in the recent Android versions.

> I don't understand the pending check

That's CI service that performs some basic checks. Not relevant for README.

A note to merger:
 - [ ] Squash commits before merge.

File: README.md (hunk 1 above)
Comment (Lokesh-Krishna) [inline README.md]: I wasn't sure if I could be upload them directly here. Is that what I should do now?

**Comment by Lokesh-Krishna** [comment]
@di72nn Thank you so much for your reply. To make these changes, I will be working on my own fork once again? How do I submit the changes as a part of this PR instead of making a new one? I am sorry but this is my first PR ever.

**Comment by di72nn** [comment]
Just push more commits to the same branch in your fork and the commits will come up here.

**Comment by Lokesh-Krishna** [comment]
I made most of the changes but I am not sure how to mention [wallabag.it](https://wallabag.it/). I mentioned Framabag because it is mentioned on the app itself and because it is a free service.  
Surely there must be other paid hosts for wallabag too?

Also, regarding the permissions, I have tried explaining it to the best of my understanding. Are there any corrections I can make?

**Comment by Lokesh-Krishna** [comment]
@di72nn I was browsing through merged pull requests and saw that wallabag.it is what will be replacing v2.wallabag.org and so have mentioned it as well in the readme now.  
Are there any more changes you'd like?

**Comment by Lokesh-Krishna** [comment]
Closing this pull request in favour of making a new one that squashes all commits into one and also so that I can get rid of my messed up master branch so that I can make better pull requests in the future.

**Pull request closed without merging**